src/common/createplay/CreatePlay.jsx

```diff
@@ -11,6 +11,21 @@ import Loader from "common/spinner/spinner";
 import { Plays } from "common/services/plays";
 import { useNavigate } from "react-router-dom";
 import { useSearchParams } from "react-router-dom";
+import { ReactComponent as NotAllowedImage } from "../../images/img-403.svg"
+
+const NoCreationInProdScreen = () => {
+  return <div className="w-full h-full flex flex-col justify-center items-center gap-8">
+    <div>
+      <NotAllowedImage width={550} />
+    </div>
+
+    <div className="text-center">
+      You can't create plays in production
+      <br />
+      <a className="text-link-default" href="https://github.com/reactplay/react-play/blob/main/CREATE-PLAY.md" target="__blank__">read this</a> for more details
```

Comment:
This is interesting, what is `__blank__` instead of `_blank`?

Reply:
Doesn't it give any warning as you are not using `rel` property?

Reply:
@koustov After a little searching, I found that `__blank__` isn't a valid target value the true one is `_blank`, and also found that the `rel` attribute isn't required in major browsers as it's automatically set to `rel="noopener"`.

- [Source](https://stackoverflow.com/a/50709724/14088437)

Reply:
Btw, I have changed the attributes to `target="_blank"` and `rel="noopener noreferrer"`.

Reply:
> @koustov After a little searching, I found that `__blank__` isn't a valid target value the true one is `_blank`, and also found that the `rel` attribute isn't required in major browsers as it's automatically set to `rel="noopener"`.
> 
> * [Source](https://stackoverflow.com/a/50709724/14088437)

I was so curious :)